addons/zeusFactory/functions/fnc_createTransport.sqf

```diff
@@ -30,11 +30,28 @@ private _driver = _group createUnit [_crewType, (getPos _vehicle), [], 0, "NONE"
 _group addVehicle _vehicle;
 _driver moveInDriver _vehicle;
 
-if (_addGunner) then {
+// add a gunner to all available turrets
+// just ace_common_fnc_getDoorTurrets but without getting FFV seats
+
+private _gunnerTurrets = [];
+private _turrets = allTurrets [_vehicle, false];
+
+{
+    private _config = configOf _vehicle;
+    _config = [_config, _x] call ACEFUNC(common,getTurretConfigPath);
+
+    if (((getNumber (_config >> "isCopilot")) == 0) && {count getArray (_config >> "weapons") > 0}) then {
+        _gunnerTurrets pushBack _x;
+    };
+    false
+} count _turrets;
```

Comment:
just fyi we've found that `...; false} count [` is actually slower than `forEach` now (it used to be faster)
I think there are some edge cases where it can be faster if you know for sure you have code that don't need the true/false at the end but it's still probably not worth it